### PR TITLE
Add deploy-4 docs

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/deploy/deploy-wso2-using/aws.md [hunk 1]
```diff
@@ -1,8 +1,4 @@
----
```
Comment (sherenewso2) [inline en/docs/deploy/deploy-wso2-using/aws.md:1]: shall we change the folder name "deploy-wso2-using" to something like "deployment-options"

File: en/docs/deploy/deploy-wso2-using/docker-desktop.md [hunk 2]
```diff
@@ -1,8 +1,4 @@
----
-template: templates/single-column.html
----
-
-# Deploying WSO2 Identity Server on Kubernetes using Docker Desktop 
+# Deploy WSO2 Identity Server on Kubernetes using Docker Desktop
```
Comment (sherenewso2) [inline en/docs/deploy/deploy-wso2-using/docker-desktop.md:1]:
```suggestion
# Deploy WSO2 Identity Server on Kubernetes Using Docker Desktop 
```

File: en/docs/deploy/deploy-wso2-using/eks.md [hunk 3]
```diff
@@ -1,8 +1,4 @@
----
-template: templates/single-column.html
----
-
-# Deploying WSO2 Identity Server on Kubernetes using AWS-EKS
+# Deploy WSO2 Identity Server on Kubernetes using AWS-EKS
```
Comment (sherenewso2) [inline en/docs/deploy/deploy-wso2-using/eks.md:1]:
```suggestion
# Deploy WSO2 Identity Server on Kubernetes Using AWS-EKS
```

Comment (sherenewso2) [reply en/docs/deploy/deploy-wso2-using/eks.md:1]: please check on other pages

File: en/docs/deploy/migrate/migrating-userstore-managers.md [hunk 4]
```diff
@@ -0,0 +1,579 @@
...
+
+## Dry run
+
+Dry-run capability in the user store migrator will allow the migration utility to validate the system for the 
+user store configurations and generate a report regarding the migration compatibility. In the generated migration report, 
+if there are any warnings in the generated migration report, it is recommended to contact the WSO2 support to identify the best migration strategy.
+
+---
+
+### How to run
```
Comment (sherenewso2) [inline en/docs/deploy/migrate/migrating-userstore-managers.md:16]: h2

File: en/docs/deploy/migrate/migrating-userstore-managers.md [hunk 5]
```diff
@@ -0,0 +1,579 @@
...
+---
+
+## Migrate userstores
+
+If you have multiple tenants and multiple user stores and you only need to migrate a few of them, refer to the 
+configuration section below. If you need to migrate all of them, use the `migrateAll` property (This is set to true by default).  
+
+---
+
+### Special scenarios
```
Comment (sherenewso2) [inline en/docs/deploy/migrate/migrating-userstore-managers.md:44]: h2

File: en/docs/deploy/migrate/migrating-userstore-managers.md [hunk 6]
```diff
@@ -0,0 +1,579 @@
...
+| tenantDomain | Domain name of the tenant. (Mandatory) |
+| increment | Number of users to be updated in each iteration. (Optional) |
+| startingPoint | Where should the migration start from (Offset). This is useful if the migration stopped in the middle and needs to restart. (Optional) |
+| scimEnabled | Whether SCIM is enabled for user stores in this tenant. (Optional) |
+| migratingDomains | List of comma-separated domain names that should be migrated to this domain. (Optional) |
+| forceUpdateUserId | Mark whether user IDs should be updated even though there is already an existing ID. (Optional) |
+
+---
+
+## Populate UserIds for the userstores
```
Comment (sherenewso2) [inline en/docs/deploy/migrate/migrating-userstore-managers.md:82]:
```suggestion
## Populate User IDs for the userstores
```

File: en/docs/deploy/migrate/migrating-userstore-managers.md [hunk 7]
```diff
@@ -0,0 +1,579 @@
...
+    CALL SYSPROC.ADMIN_CMD('REORG TABLE UM_USER')
+    /
+
+    UPDATE UM_USER SET UM_USER_ID = NEWUUID()
+    /
+    CALL SYSPROC.ADMIN_CMD('REORG TABLE UM_USER')
+    /
+    ```
+
+2.  If you have JDBC secondary user stores with SCIM enabled, execute the following queries on the UM database.
```
Comment (sherenewso2) [inline en/docs/deploy/migrate/migrating-userstore-managers.md:165]: check places where user stores are two words

File: en/mkdocs.yml [hunk 8]
```diff
@@ -361,6 +361,20 @@ nav:
...
+    - 'Migrate':
+      - 'Migration Process': deploy/migrate/migration-process.md
+      - 'What has Changed': deploy/migrate/migrating-what-has-changed.md
+      - 'Migration Guide':
+        - 'Before you Begin': deploy/migrate/migration-guide.md
+        - 'Prepare for Migration': deploy/migrate/migrating-preparing-for-migration.md
+        - 'Migrate to 5.11.0': deploy/migrate/migrating-to-5110.md
+        - 'Migrate Userstore Managers': deploy/migrate/migrating-userstore-managers.md
+      - 'Feature Deprecation': deploy/migrate/wso2-identity-server-feature-deprecation.md
+    - 'Deploy WSO2 Using':
```
Comment (sherenewso2) [inline en/mkdocs.yml:373]: Shall we change this topic to something like "Deployment Options" or a more suitable topic

**Review by sherenewso2 — APPROVED** [review]

File: en/docs/deploy/migrate/migrating-userstore-managers.md (hunk 4 above)
Comment (gomathyk) [reply en/docs/deploy/migrate/migrating-userstore-managers.md:16]: It's a sub heading under "Dry Run"

**Pull request merged**